python/ray/tune/syncer.py

```diff
@@ -83,6 +83,7 @@ def log_sync_template(options=""):
 
     rsh = "ssh -i {ssh_key} -o ConnectTimeout=120s -o StrictHostKeyChecking=no"
     rsh = rsh.format(ssh_key=quote(ssh_key))
+    options += " --exclude='checkpoint_tmp*'"
```

Comment:
May be there should be one project-wide rsync-filter file for checkpoints because rsync looks to be used elsewhere.